### ISSUE TITLE
Hyperparameter tuning and implementation of training and validation loss curve

### ISSUE THREAD

**Issue by ShawShank-Devmption** [csivitu/Bird-Trajectory-Prediction#6]
- Verify convergence and detect overfitting visually.
- The model currently uses fixed hyperparameters. Improve the performance by experimenting with the params like learning rate schedules, hidden dim, seq_len etc.
